CHANGELOG.md

```diff
@@ -1,4 +1,16 @@
 ## 3.68.0 (Unreleased)
+FEATURES:
+* **New Resource:** pubsub: Added schema resource as `google_pubsub_schema` to google pubsub ([#9116](https://github.com/hashicorp/terraform-provider-google/pull/9116))
```

Comment:
Typically for new resources, we'll just include the resource name (and sometimes a description)

The extra text doesn't add much here, I'd recommend removing it.

Reply:
done